packages/oscal-react-library/src/components/OSCALCatalogGroup.js

```diff
@@ -133,59 +216,85 @@ function OSCALCatalogControlListItem(props) {
         urlFragment={urlFragment}
         fragmentPrefix={appendToFragmentPrefix(fragmentPrefix, control.id)}
         fragmentSuffix={shiftFragmentSuffix(fragmentSuffix)}
+        previousHandledFragment={previousHandledFragment}
+        setPreviousHandledFragment={setPreviousHandledFragment}
       />
     </CollapsibleListItem>
-  ) : (
-    <StyledListItemPaper>
-      <StyledListItem>
-        <WithdrawnListItemText primary={itemText} withdrawn={withdrawn} />
-      </StyledListItem>
-    </StyledListItemPaper>
   );
 }
 
 function OSCALCatalogGroupList(props) {
```

Comment:
Again, this would be a really nice place for an `interface`. Can we at least add some `jsdoc`?

Reply:
I'm wondering to what extent we want to add JSDocs? Are we adding them just to our utility functions or are we including OSCAL components in these now?

Reply:
> I've added JSDoc to all OSCALCatalogGroup sub-components.

@Bronstrom For clarity, I was mostly looking for the things under `props` to be documented. `props` contains a lot of fields and, as someone reviewing, it is a bit difficult to know what they are. 

As an example, this is what we  to do in `OSCALBackMatter.tx`. Each `prop` being passed needs an associated type and ideally a description. Having those types and descriptions now in some type of comment, would really help a lot in both reviewing this code and eventually switching to `.tsx`.

https://github.com/EasyDynamics/oscal-react-library/blob/6dc2a5370d5d8cdc7b865bc84d83106f3f25923f/packages/oscal-react-library/src/components/OSCALBackMatter.tsx#L87-L97

If it is a big thing to add, we can skip for now.

Reply:
I've added documentation for the props. I can't directly place the documentation above each destructured property (the documentation is just above the `const`) and it's not going to look as nice as the JSDoc above a function, but I think it should give a better idea of the types for TS implementation.

Reply:
> not going to look as nice as the JSDoc

Yeah I agree it looks a bit nasty :smile:. Hopefully it will motivate us to switch to file to `.tsx` soon! Thank you for humoring me.